tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: install cron (bionic support)
+  apt: name={{ item }} update_cache=yes state=present
```

Comment:
Can you add `cache_valid_time={{ 24 * 60 * 60 }}` so that if the cache has been updated within the past 24 hours, it won't be updated again?

Reply:
This value would replace update_cache, correct?

Reply:
Correct.

Reply:
@adborden this has been updated.